pages/_app.js

```diff
@@ -42,6 +42,11 @@ export default class HomePage extends App {
             name="viewport"
             content="width=device-width, initial-scale=1.0"
           />
+          <link
+            rel="shortcut icon"
+            type="image/x-icon"
+            href="../static/imgs/favicon.ico"
```

Comment:
This favicon is different from the favicon used on catho's home(www.catho.com.br). Was it suggested to use it??